code/Player/MainPlayer.cs

```diff
@@ -112,25 +112,17 @@ public override void OnKilled()
             Gamemode.Instance.RoundManager.CurrentRound.OnPlayerKilled(this);
         }
 
-        public override void PostCameraSetup(ref CameraSetup setup)
+        protected override void OnDestroy()
         {
-            base.PostCameraSetup(ref setup);
-            Inventory.ViewModel.PostCameraSetup(ref setup);
+            base.OnDestroy();
```

Comment:
Should be at the end ??

Reply:
I don't think that creates any unexpected behaviour for the moment, I don't know which is more logical